TIMING_AND_MOBILE_FIXES.md

```
@@ -0,0 +1,180 @@
+- `App.jsx`: 
+  - Fallback timer: 7000ms → 6000ms
+  - Background reveal: 3500ms → 2500ms
+
+#### New Timeline:
+```
+0.0s - Loader appears
+0.1s - Card fades in (0.6s duration)
+0.7s - Card fully visible
+1.0s - Overlay fades in (0.5s duration)
+1.5s - Overlay visible
+1.8s - Expansion begins (1.0s duration)
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)